apluslms_roman/docker_cache.py

```diff
@@ -0,0 +1,9 @@
+from .cache_file import CacheFile
+
+class DockerCache(CacheFile):
```

Comment:
You could have this in start of backend/docker.py. Otherwise, this would be `backends/docker/cache.py`, which would require moving `backends/docker.py` to `backends/docker/__init__,py`